srcpkgs/mongoose/template

```diff
@@ -1,29 +1,40 @@
 # Template file for 'mongoose'
 pkgname=mongoose
-version=6.18
-revision=4
+version=7.6
+revision=1
 build_style=gnu-makefile
-make_use_env=compliant
-makedepends="openssl-devel"
-short_desc="Easy to use Web server"
-maintainer="Orphaned <orphan@voidlinux.org>"
+makedepends="$(vopt_if openssl openssl-devel mbedtls-devel)"
+short_desc="Embedded Networking Library with TCP,UDP,HTTP,MQTT,WebSocket and SSL/TLS"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="GPL-2.0-only"
-homepage="https://cesanta.com/"
+homepage="https://mongoose.ws/"
 distfiles="https://github.com/cesanta/mongoose/archive/${version}.tar.gz"
-checksum=f5c10346abc9c72f7cac7885d853ca064fb09aad57580433941a8fd7a3543769
+checksum=1ef09d971b6de1a6317c109980d6fb5a9c19b39efef2506d6b76869644b3dafa
+
+make_use_env=yes
+make_build_target=linux-libs
+make_check_target=test
+make_check=no # CBA for now
```

Comment:
1. "CBA" isn't really a good reason to skip checks.
2. If there is a valid reason to skip checks, there is no value in defining `make_check_target` above.

Reply:
That's why I marked this as draft. I'll be fixing it.

Reply:
fixed.